packages/block-editor/src/components/block-list/block.js

```diff
@@ -400,13 +400,27 @@ function BlockListBlock( {
 		className
 	);
 
+	// By default, a block is focusable, which enables it to be set as the
+	// selected block when the user interacts within its content. Through
+	// assignment of its wrapper props, however, a block can opt-out of this
+	// behavior to effectively treat the block as a "pass-through", one which
+	// is not intended to be directly selected under normal circumstances.
+	// Often this occurs in the context of deep block nesting, where focus
+	// selection instead occurs by one of the ancestor blocks.
+	let isFocusable = true;
+
 	// Determine whether the block has props to apply to the wrapper.
 	let blockWrapperProps = wrapperProps;
 	if ( blockType.getEditWrapperProps ) {
 		blockWrapperProps = {
 			...blockWrapperProps,
 			...blockType.getEditWrapperProps( attributes ),
 		};
+
+		isFocusable = ! ( 'tabIndex' in blockWrapperProps ) || (
+			isFinite( blockWrapperProps.tabIndex ) &&
+			Number( blockWrapperProps.tabIndex ) !== '-1'
```

Comment:
![Screen Shot 2019-06-10 at 14 34 34](https://user-images.githubusercontent.com/699132/59195701-e8dd2480-8b8c-11e9-91f7-e95245990b63.png)

Based on the above, it seems like this should be:

```suggestion
			Number( blockWrapperProps.tabIndex ) !== -1
```

Reply:
> ![Screen Shot 2019-06-10 at 14 34 34](https://user-images.githubusercontent.com/699132/59195701-e8dd2480-8b8c-11e9-91f7-e95245990b63.png)
> 
> Based on the above, it seems like this should be:

Oops, good catch. At first, I'd implemented this as `blockWrapperProps.tabIndex !== '-1'` and later considered to make it more durable by capturing values passed either as string or number. I clearly forgot to update the `'-1'` though.